src/TitleEdit/TitleEditContainer.js

```diff
@@ -69,14 +70,18 @@ function TitleEditContainer({ history, location, match, mutator }) {
           });
           setTimeout(onCancel);
         })
-        .catch(() => showCallout({
-          messageId: 'ui-receiving.title.actions.save.error',
-          type: 'error',
-          values: {
-            title: newTitle.title,
-            poLineNumber: line.poLineNumber,
-          },
-        }));
+        .catch(async (response) => {
+          const errorCode = await handleErrorResponse(response);
+
+          showCallout({
+            messageId: `ui-receiving.title.actions.save.error.${errorCode}`,
```

Comment:
there is a possibility that back-end returns error code that isn't in translations. An ugly toast with translation id will appear.

Reply:
I'll add default message for it